### ISSUE TITLE
Not working on Windows

### ISSUE THREAD

**Issue by entozoon** [adamwalzer/vscode-scss-lint#22]
Hi there, love the plugin but doesn't seem to work currently.
I have:
``` scss-lint 0.57.0 ``` (most recent)
```
Error running scss-lint
Unable to open '\www\project-name\src\scss\\.scss-lint.yml': A system error occurred (UNKNOWN: unknown error, stat '\\www\project-name\src\scss\.scss-lint.yml').
```

This happens even when the `.scss-lint.yml` file is empty. It is definitely finding it though - because if I delete it, there is no linting or error at all.

Any ideas?

**Comment by adamwalzer** [comment]
@entozoon thanks for the report. It seems like this issue is likely with the double \ before .scss-lint.yml. Are you setting any configs? This could very well be my error because it's rather tough for me to test on windows at this time.

**Comment by adamwalzer** [comment]
@entozoon I made an update that I believe will resolve your issue. Please let me know if this issue is resolved in version 0.1.6. Thanks!

**Comment by entozoon** [comment]
That seems to have done the trick mate! Thanks very much
